lib/test-worker.js

```diff
@@ -30,6 +30,23 @@ process.installPrecompilerHook();
 const dependencies = [];
 process.installDependencyTracking(dependencies, testPath);
 
+if (opts.extensions) {
+	opts.extensions.forEach(function (ext) {
+		switch (ext) {
+			case 'ts':
+				var extTs = require('./ext/ts');
+				extTs.register();
+				break;
+			case 'js':
+				// We already have babel for ES6/7
+				break;
+
+			default:
+				throw new Error('AVA: `--extension=' + ext + '` is not supported by AVA.');
+		}
+	});
+}
```

Comment:
Please remove these changes. Right now I'd prefer our TypeScript support is consistent with our Babel support, where dependencies are not automatically transpiled.

I know that's not ideal but we can revisit it later.